src/client/app/redux/selectors/threeDSelectors.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { createSelector } from '@reduxjs/toolkit';
+import {
+	selectThreeDMeterOrGroup, selectThreeDMeterOrGroupID
+} from '../slices/graphSlice';
+import { selectGroupDataById } from '../../redux/api/groupsApi';
+import { MeterOrGroup } from '../../types/redux/graph';
+import { AreaUnitType } from '../../utils/getAreaUnitConversion';
+import { selectMeterDataById } from '../../redux/api/metersApi';
+import { selectNameFromEntity } from './plotlyDataSelectors';
+
+
+// Memoized Selectors
+export const selectThreeDComponentInfo = createSelector(
+	selectThreeDMeterOrGroupID,
+	selectThreeDMeterOrGroup,
+	selectMeterDataById,
+	selectGroupDataById,
+	(id, meterOrGroup, meterDataById, groupDataById) => {
+		//Default Values
+		let meterOrGroupName = 'Unselected Meter or Group'
+		let isAreaCompatible = true;
```

Comment:
My reading of the code indicates this must be set by selectNameFromEntity so this value does not matter. Wanted to verify that is correct. If so, I think we should set to empty string (or comment).

Reply:
This is correct. Not is an empty string instead.